### PR TITLE
CAKE-5532 | Add Translations for Disclaimer Message

### PR DESCRIPTION
**Pull request by cschucode**
Add support for translating disclaimer message for:
- Affiliate Service module
- Watch Show module

We currently only have translations for ES, DE, and IT.

### CODE REVIEW

File: extensions/wikia/AffiliateService/AffiliateServiceHooks.class.php [hunk 1]
```diff
@@ -83,6 +83,9 @@ public static function onWikiaSkinTopScripts( Array &$vars, &$scripts ) {
 		// only enable it onNS 0 AND dev env
 		$vars['wgAffiliateEnabled'] = $isProperNamespace && $isProperCommunity;
 
+		// add translations from AffiliateService.i18n.php
+		$vars['disclaimerMessage'] = wfMessage('affiliate-service-disclaimer')->escaped();
```
Comment (mklucsarits) [inline extensions/wikia/AffiliateService/AffiliateServiceHooks.class.php]: `escaped()` is intended to escape HTML and avoid XSS. You can just use `->plain()` instead.

File: extensions/wikia/WatchShow/WatchShowService.class.php [hunk 2]
```diff
@@ -20,6 +20,7 @@ public function index() {
 			// TODO:: Replace $wgWatchShowImageURLMobileDarkTheme with $wgWatchShowImageURLDarkTheme variable once set in Wiki Factory
 			'imageURL' => SassUtil::isThemeDark() ? $wgWatchShowImageURLMobileDarkTheme : $wgWatchShowImageURL,
 			'trackingPixelURL' => $wgWatchShowTrackingPixelURL,
+			'disclaimerMessage' => wfMessage( 'watch-show-disclaimer' )->escaped(),
```
Comment (mklucsarits) [inline extensions/wikia/WatchShow/WatchShowService.class.php]: Same as above

**Review by JoshuaRogan — APPROVED** [review]

File: extensions/wikia/AffiliateService/AffiliateService.i18n.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
+<?php
+$messages = array();
```
Comment (JoshuaRogan) [inline extensions/wikia/AffiliateService/AffiliateService.i18n.php:2]: I would consult confluence (maybe: https://wikia-inc.atlassian.net/wiki/spaces/PLAT/pages/110593849/i18n) or ping the engineering channel to determine the conventions for these files.

File: extensions/wikia/AffiliateService/AffiliateService.i18n.php [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+$messages['en'] = array(
+	'affiliate-service-disclaimer' => 'Fandom may earn an affiliate commission on sales made from links on this page.',
+);
+
+$messages['qqq'] = array(
```
Comment (JoshuaRogan) [inline extensions/wikia/AffiliateService/AffiliateService.i18n.php:8]: Is this right? qqq?

Comment (cschucode) [reply extensions/wikia/AffiliateService/AffiliateService.i18n.php:8]: I did have a question about that. I'm wondering if it's a default fallback.

Comment (KockaAdmiralac) [reply extensions/wikia/AffiliateService/AffiliateService.i18n.php:8]: `qqq` is a dummy language used for message descriptions, it does not actually display in the interface.

Comment (cschucode) [reply extensions/wikia/AffiliateService/AffiliateService.i18n.php:8]: It's a way of documenting messages.

https://www.mediawiki.org/wiki/Localisation#Message_documentation

File: extensions/wikia/AffiliateService/AffiliateService.i18n.php (hunk 3 above)
Comment (cschucode) [reply extensions/wikia/AffiliateService/AffiliateService.i18n.php:2]: Following the convention of other i18n files in the repo including Trivia Quizzes.

**Pull request merged**